cterasdk/common/utils.py

```diff
@@ -0,0 +1,63 @@
+import re
+import logging
+
+from .object import Object
+
+
+def union(g1, g2):
+    """
+    Create a union of two lists, without duplicates.
+
+    :return list: the merged dictionary
+    """
+    return g1 + list(set(g2) - set(g1))
+
+
+class BaseObjectRef(Object):
+    """
+    Class Representing a Portal Base Object Reference
+
+    :ivar str classname: Base object class name
+    :ivar str uid: Base object unique identifier (uid)
+    :ivar str tenant: Base object tenant name
+    :ivar str name: Base object name
+    """
+    def __init__(self, classname, uid, tenant, name):
+        """
+        :param str classname: Base object class name
+        :param str uid: Base object unique identifier
+        :param str tenant: Base object tenant
+        :param str name: Base object name
+        """
+        self.classname = classname
+        self.uid = uid
+        self.tenant = tenant
+        self.name = name
+
+    def in_tenant_context(self):
+        """
+        Returns True if the base object exists in a context of a tenant
+        """
+        return bool(self.tenant)
+
+    def __str__(self):
+        """
+        Returns a string representation of the base object reference
+        """
+        return '/'.join(['objs', self.uid, self.tenant, self.classname, self.name])
+
+
+def parse_base_object_ref(base_object_ref):
+    """
+    Parse a base object reference.
+
+    :return cterasdk.common.utils.BaseObjectRef param: Object holding the classname, uid, tenant and object name
```

Comment:
Move param above return

Reply:
Hi Ygal, I believe you mean add a 'param' entry to the documentation. Have a look at the latest commit

Reply:
Yes that's what I meant :)